### PR TITLE
chore: e2e optimizations

### PR DESCRIPTION
**Pull request by mroz22**

### CODE REVIEW

File: docker/docker-compose.suite-base.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ services:
     volumes:
       - ..:/usr/src/service
       - ../node_modules:/usr/src/service/node_modules
+      - /mnt/wps:/mnt/wps
```
Comment (mroz22) [inline docker/docker-compose.suite-base.yml:12]: this is required for local development after going away from next

File: packages/integration-tests/package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
         "@cypress/skip-test": "^2.6.0",
         "@cypress/webpack-preprocessor": "^5.5.0",
         "cypress-image-snapshot": "^4.0.0",
+        "chrome-remote-interface": "^0.31.0",
```
Comment (mroz22) [inline packages/integration-tests/package.json:19]: this allows us to use chrome dev tools from tests. for example to force state of dom element to :hover

File: packages/integration-tests/projects/suite-web/tests/suite/analytics.test.ts [hunk 3]
```diff
@@ -51,6 +51,7 @@ describe('Analytics', () => {
 
         // important, suite needs time to save initialRun flag into storage
         cy.getTestElement('@suite/loading').should('not.exist');
+        cy.discoveryShouldFinish();
 
         // go to settings
         cy.wait(5000);
```
Comment (mroz22) [inline packages/integration-tests/projects/suite-web/tests/suite/analytics.test.ts]: ?

File: packages/integration-tests/projects/suite-web/plugins/index.js [hunk 4]
```diff
@@ -12,7 +13,25 @@ import * as metadataUtils from '../../../../suite/src/utils/suite/metadata';
 
 const webpackPreprocessor = require('@cypress/webpack-preprocessor');
 
+function ensureRdpPort(args) {
```
Comment (tomasklim) [inline packages/integration-tests/projects/suite-web/plugins/index.js]: nit: arrow function?

File: packages/integration-tests/projects/suite-web/plugins/index.js [hunk 5]
```diff
@@ -218,5 +260,38 @@ module.exports = on => {
...
+                client = null;
+            }
+
+            return Promise.resolve(true);
+        },
+        activatePrintMediaQuery: async () => {
+            client = client || (await CDP({ port }));
+
+            return client.send('Emulation.setEmulatedMedia', { media: 'print' });
+        },
```
Comment (tomasklim) [inline packages/integration-tests/projects/suite-web/plugins/index.js]: I think that we will never test/implement it as probably no one will print Suite on the paper 😆

**Review by vdovhanych — APPROVED** [review]

**Review by tomasklim — CHANGES_REQUESTED** [review]

File: packages/integration-tests/projects/suite-web/plugins/index.js (hunk 5 above)
Comment (vdovhanych) [reply packages/integration-tests/projects/suite-web/plugins/index.js]: what about transaction export and printig that ? 😆

Comment (mroz22) [reply packages/integration-tests/projects/suite-web/plugins/index.js:275]: probably not :wink: haha.

Comment (mroz22) [reply packages/integration-tests/projects/suite-web/plugins/index.js]: not really needed, I removed that.

**Review by tomasklim — APPROVED** [review]
LGTM

**Pull request merged**